react/features/stream-effects/virtual-background/JitsiStreamBackgroundEffect.js

```diff
@@ -103,6 +103,12 @@ export default class JitsiStreamBackgroundEffect {
             this._outputCanvasCtx.filter = 'blur(8px)';
         }
 
+        // save current context before applying transformations
+        this._outputCanvasCtx.save();
```

Comment:
Are you flipping the image automatically for the image based virtual backgrounds as well ? Isn't this supposed to be done only when desktop share is selected for the background ?

Reply:
I think that we need to apply it to all cases. I'm just thinking that: - in some cases the user wants to put something specific on his background and explain to other users, for example, significations of his background.

Reply:
You're right. Fixed. Isolated the desktop share case.